lib/omise/charge.ex

```diff
@@ -254,12 +257,15 @@ defmodule Omise.Charge do
   ## Examples
 
       Omise.Charge.capture("chrg_test_4xso2s8ivdej29pqnhz")
+      Omise.Charge.capture("chrg_test_4xso2s8ivdej29pqnhz", key: "skey_test_123", capture_amount: 3000)
+      Omise.Charge.capture("chrg_test_4xso2s8ivdej29pqnhz",[key: "skey_test_123",capture_amount: 3000])
```

Comment:
We probably don't need this line. It's the same as the example on line 260, which is written in a more typical - idiomatic - style.

Also, do you want to add any information about the partial capture to the doc description (before the `NOTE` on line 253)?